source/code-snippets/collation/query-index-collation.js

```diff
@@ -1,14 +1,30 @@
 // start query index collation
+/*
+   Retrieve documents that match the "year" value "1980"
+   in descending order of the value of the "title" field,
+   specifying a collation that uses the index.
+*/
 myColl.find({"year": 1980}, {"collation" : {"locale" : "en_US" }})
   .sort({"title": -1});
 // end query index collation
 
 // start query without index collation
 // no collation specified
+/*
+   Retrieve documents that match the "year" value "1980"
+   in descending order of the value of the "title" field
+   that does not use the collation index.
+*/ 
```

Comment:
Q: For this code comment and the next, is there a way to combine the code comments content so these aren't stacked?